simplq/src/store/asyncActions/getUserQueues.js

```diff
@@ -0,0 +1,46 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import useAuth, { makeAuthedRequest } from 'api/auth';
+import * as RequestFactory from 'api/requestFactory';
+
+const typePrefix = 'getUserQueues/requestStatus';
+
+/**
+ * A hook to access the fetchQuees async action creator.
+ *
+ * @returns — fetchQueues async async action creator
+ *
+ * @example
+ *
+ * import useFetchQueues from 'store/queues'
+ *
+ * const fetchQueues = useFetchQueues()
+ */
+const useGetUserQueues = () => {
+  const auth = useAuth();
+
+  const getUserQueues = createAsyncThunk(typePrefix, async () => {
+    if (!auth || !auth.isAuthenticated) {
```

Comment:
> We definitely do not need !auth, it is guaranteed to be there by the hook. There was a lot of copy-paste happening so there will be a lot to remove 😊 I haven't explored what exactly should happen when not authenticated, so that should be implemented later.

let's add a comment here for my reference:
```
// TODO: Remove `!auth.isAuthenticated` after resolving https://github.com/SimplQ/simplQ-frontend/issues/477 
```

also lets remove `!auth` and add `auth.isLoading` to the check

Reply:
Removing it everywhere. All checks of the status should be done in reducers triggered by `.pending``.fulfilled`, ` and `.rejected` actions.